WISE-Community/WISE
# Fixed bug editing note in preview

How to test:
1. Create a note in preview
2. Edit the note.

You should be able to make changes to the note.
